modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -25,14 +28,46 @@ The `high-limit` format allows more nodes and relationships to be stored than `s
 
 You can either set the store format when creating a new database or change the store format of an existing database.
 
+The default store format of new databases is `aligned` and is controlled by the xref:configuration/configuration-settings.adoc#config_db.format[`db.format`] configuration setting in the _neo4j.conf_ file.
```

Comment:
Why mention the default value when we link to the setting. I don't think we do that for any other setting in the documentation. Also, its easy to forget to update this page when we update the default value in the future

Reply:
My idea was to tell the users what the default format is for all new databases and how to change it if they want.